scenarios/common/infrastructure_deployment.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import hashlib
+import os
+import time
+import uuid
+import shutil
+
+from python_terraform import Terraform
+
+execution_id = str(uuid.uuid1())[:8]
+
+
+def is_infra_deployment_node(config):
+    if hasattr(config, 'slaveinput'):

```

Comment:
use workerinput not slaveinput the later is deprecated as part of a general movement to being more pc in code.